### PR TITLE
Fixed Some typos and Grammatical errors.

### PR DESCRIPTION
**Pull request by marooncoder09**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -7,29 +7,28 @@ Solidity
     :align: center
 
 Solidity is an object-oriented, high-level language for implementing smart
-contracts. Smart contracts are programs which govern the behaviour of accounts
+contracts. Smart contracts are programs that govern the behavior of accounts
```
Comment (hrkrshnn) [inline docs/index.rst:10]: About behavior v/s behaviour. The docs use British spelling.

**Review by hrkrshnn — COMMENTED** [review]
Generally looks good. Will take another look after the release.

**Comment by chriseth** [comment]
Thanks for your pull request! Please note that we do not use the Oxford comma and we use British spelling unless there is an alternative that is valid in both American and British English. Furthermore, please try to make the changes in a way that do not introduce or erase line breaks, since that makes it much easier to review your changes.

File: docs/index.rst (hunk 1 above)
Comment (marooncoder09) [reply docs/index.rst:10]: ok cool 😅

**Comment by hrkrshnn** [comment]
@marooncoder09 could you please revert the American English words and perhaps the oxford comma as well? The rest of the changes look good, and we can merge it. Thanks.

**Pull request closed without merging**